src/Ot2Rec/metadata.py

```diff
@@ -239,6 +243,22 @@ def get_mc2_temp(self):
         self.metadata['frame_dose'] = df.frame_dose.to_list()
 
 
+    def get_acquisition_settings(self):
+        df = pd.DataFrame(self.metadata)
+        base_folder = '/'.join(df.file_paths.values[0].split('/')[:-1])
+
+        ts = list(set(df.ts))[0] # Assuming settings same across one data set
```

Comment:
this is probably fine for now but for the future might need to add the option for different settings